neutron_plugin/port.py

```diff
@@ -95,6 +103,52 @@ def _port_update(neutron_client, port_id, args, ext_port):
     runtime_properties[MAC_ADDRESS_PROPERTY] = ext_port['mac_address']
 
 
+def _get_ips_from_port(port):
+    """
+    This method will extract ipv4 & ipv6 for from port object
+    :param port:
+    :return:
+    """
+    net_ips = port['fixed_ips'] if port.get('fixed_ips') else []
+    ips_v4 = []
+    ips_v6 = []
+    for net_ip in net_ips:
+        if net_ip.get('ip_address'):
+            ip_address = net_ip['ip_address']
+            try:
+                # Lookup all ipv4s
+                IP(ip_address, ipversion=4)
+                ips_v4.append(ip_address)
+            except ValueError:
+                # If it is not an ipv4 then collect the ipv6
+                IP(ip_address, ipversion=6)
+                ips_v6.append(ip_address)
```

Comment:
Should we try to except this?

Reply:
Usualy fixed_ips contains a lot of stuff but since we are getting only `ip_address` its confirmed that either its a ipv4 or ipv6 , what do you think, if reached here then its ipv6 for sure